resource/read.go

```diff
@@ -14,6 +15,9 @@ type ReadRequest struct {
 	// operation.
 	State tfsdk.State
 
+	// Private is resource private state data.
```

Comment:
```suggestion
	// Private is provider-defined resource private state data which was previously
	// stored with the resource state. This data is opaque to Terraform and does
	// not affect plan output. Any existing data is copied to
	// ReadResourceResponse.Private to prevent accidental private state data loss.
	//
	// Use the GetKey method to read data. Use the SetKey method on
	// ReadResourceResponse.Private to update or remove a value.
```